plugins/LocalContainerProvider/LocalContainerProvider.py

```diff
@@ -105,26 +119,26 @@ def saveContainer(self, container: "ContainerInterface") -> None:
     #   \param container_id The ID of the container to load the metadata of.
     #   \return The metadata of the specified container, or ``None`` if the
     #   metadata failed to load.
-    def loadMetadata(self, container_id: str) -> Optional[Dict[str, Any]]:
+    def loadMetadata(self, container_id: str) -> Dict[str, Any]:
         registry = ContainerRegistry.getInstance()
         if container_id in registry.metadata:
             return registry.metadata[container_id]
 
         filename = self._id_to_path[container_id] #Raises KeyError if container ID does not exist in the (cache of the) files!
         clazz = ContainerRegistry.mime_type_map[self._id_to_mime[container_id].name]
 
-        requested_metadata = None
+        requested_metadata = {}  # type: Dict[str, Any]
```

Comment:
Don't think we need a typing here, right? Since the typing is already defined for the return value. Also because in two cases below we return an empty dict without typing.

Reply:
Unfortunately, without it, I got this:
```
plugins/LocalContainerProvider/LocalContainerProvider.py:130: error: Need type annotation for 'requested_metadata'
```

Reply:
Weird. Wonder why it's needed there but not when returning the empty dict.